samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -21,6 +21,8 @@ metadata:
   name: details
   labels:
     app: details
+  annotations:
```

Comment:
Is it okay to have empty annotations? i.e., if a customer wants to deploy this bookinfo.yaml as it is now (without running any script to put GCP identity), would this cause any problems?